### ISSUE TITLE
Error Domain=com.apple.AuthenticationServices.AuthorizationError Code=1000 \"(null)\"

### ISSUE THREAD

**Issue by luokunman** [vietaapp/node-mac-sign-in-with-apple#4]
Does electron work in a development environment?

**Comment by jpalumickas** [comment]
@luokunman yes, it should work, but you need to build with correct entitlements.

**Comment by luokunman** [comment]
But I have gathered a lot of information that entitlements can only be recognized by the electron-builder, and the development environment seems to be nowhere to be recognized

**Comment by luokunman** [comment]
> @luokunman yes, it should work, but you need to build with correct entitlements.

But I have gathered a lot of information that entitlements can only be recognized by the electron-builder, and the development environment seems to be nowhere to be recognized

After entitlements.plist is added, the application cannot be opened.

![image](https://user-images.githubusercontent.com/41885648/229450192-b133137c-c50c-4d5a-bb70-565ac9e424a5.png)

![image](https://user-images.githubusercontent.com/41885648/229450228-52337253-af1a-4f7e-b6fe-3a6b9d0b76ea.png)

**Comment by jpalumickas** [comment]
I think if you add entitlements, you need to sign it with development key with electron builder
